dubbo-metadata/dubbo-metadata-report-redis/src/main/java/org/apache/dubbo/metadata/store/redis/RedisMetadataReport.java

```diff
@@ -209,4 +228,266 @@ private String getMetadataStandalone(BaseMetadataIdentifier metadataIdentifier)
             throw new RpcException(msg, e);
         }
     }
+
+    @Override
+    public boolean registerServiceAppMapping(
```

Comment:
Store class and application names using Redis hashes
key: dubbo:mapping
field: class
value: application_names